configure.py

```diff
@@ -685,7 +685,9 @@ def set_tf_cunn_version(environ_cp):
       ldconfig_bin = which('ldconfig') or '/sbin/ldconfig'
       cudnn_path_from_ldconfig = run_shell([ldconfig_bin, '-p'])
       cudnn_path_from_ldconfig = re.search('.*libcudnn.so .* => (.*)',
-                                           cudnn_path_from_ldconfig).group(1)
+                                           cudnn_path_from_ldconfig)
+      if cudnn_path_from_ldconfig != None:
```

Comment:
[nit]: remove " != None"

Reply:
@yifeif Sorry, I can't see the comments, could you have a check please?

Reply:
Could you remove the "!= None"?